data/mods/More_Survival_Tools/armor.json

```diff
@@ -0,0 +1,120 @@
+[
```

Comment:
Most items are in this one file, I should've taken the time to separate more than just the armor.